### PR TITLE
ENDOC-164 Update screenshots for 6.3

### PR DESCRIPTION
**Pull request by nshaw**

### CODE REVIEW

**Comment by nshaw** [comment]
@joewhite101 I need the final build for a couple more images waiting on FE bug fixes but after I did the analysis I realized I could do the rest. I took the opportunity to simplify a few areas and clean up some language. It turns out the Publish Content tutorial has complete start-to-finish page publishing steps so we should keep that in mind when we get to the Pages docs.

This is published to my staging site - https://nshaw.github.io/next/docs/

File: vuepress/docs/next/tutorials/cms/digital-assets-tutorial.md [hunk 1]
```diff
@@ -1,30 +1,23 @@
...
 
-## How to create a Digital Asset
+## Create a Digital Asset
 
-To manage the Digital Assets through the Web CMS integrated with App
-Builder application
+To manage the Digital Assets through the WCMS:
 
-- From App Builder, go to `Content > Types`
+- Go to `Content > Types`
```
Comment (joewhite101) [inline vuepress/docs/next/tutorials/cms/digital-assets-tutorial.md:20]: A very very small thing but we should standardize how we're representing that menu transition. I've been doing "->" this one has ">", I've seen "-", and I've also done "-->".

Not something to fix now but a note

Comment (nshaw) [reply vuepress/docs/next/tutorials/cms/digital-assets-tutorial.md:20]: Good call, it's all over the map. I'm fine to standardize on -> if you prefer it and can keep an eye open for it as I'm updating docs.

**Review by joewhite101 — APPROVED** [review]
LGTM

**Pull request merged**